practical-07/src/components/GalleryCard.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+    <div class="edit-delete-btn">
+      <p type="button" class="btn-edit" @click="editCarDetails(carId)"><i class="fa fa-pencil"></i></p>
+      <p type="button" class="btn-delete" @click="dataDelete">x</p>  
+    </div>
+    <div class="carName">{{ carName }}</div>
+    <div class="col-auto mb-2 card-animation"><img :src="carImage" alt="car-image" /></div>
+    <div class="mb-2">
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
  <div class="mb-2">
```